gaston/second-assignment/index.js

```diff
@@ -0,0 +1,34 @@
+let arrHeroes = [];
+
+function getData() {
+  const url = "/marvel.json";
+  fetch(url)
+    .then((res) => res.json())
+    .then((res) => {
+      arrHeroes = res.data.results;
+      console.log(arrHeroes);
```

Comment:
Maybe it's better if you remove the console.log once you know the code works

Reply:
Fixed